doc/source/data/api/dataset.rst

```diff
@@ -4,7 +4,16 @@ Dataset API
 ==============
 
 .. include:: ray.data.Dataset.rst
-.. include:: ray.data.Schema.rst
+
+
+Schema
+------
+.. currentmodule:: ray.data
+
+.. autoclass:: Schema
+    :members:
```

Comment:
This documents the properties of the class

Reply:
before
<img width="946" alt="Screenshot 2025-05-16 at 4 58 21 PM" src="https://github.com/user-attachments/assets/72dc8e89-0e06-4eb7-bb32-e1853db623da" />